api/src/main/java/org/openmrs/api/OrderService.java

```diff
@@ -812,4 +812,26 @@ public Order discontinueOrder(Order orderToDiscontinue, String reasonNonCoded, D
 	 */
 	@Authorized({ PrivilegeConstants.EDIT_ORDERS, PrivilegeConstants.ADD_ORDERS })
 	public OrderGroup saveOrderGroup(OrderGroup orderGroup) throws APIException;
+	
+	/**
+	 * Fetches all order groups for the specified patient
+	 * 
+	 * @param patient the patient to match on
+	 * @return list of matching OrderGroup * @since 2.3
+	 * @throws APIException
+	 */
+	@Authorized(PrivilegeConstants.GET_ORDERS)
+	public List<OrderGroup> getOrderGroupsByPatient(Patient patient) throws APIException;
+	
+	/**
+	 * Fetches all order groups for the specified encounter
+	 *
+	 * @param encounter the encounter to match on
+	 * @return list of matching OrderGroups
+	 * @since 2.3
```

Comment:
Are you sure version 2.3 will have these changes?

Reply:
@dkayiwa i think so because in the ticket description the fix version is 2.3.0

Reply:
Looks like that version is being released without this fix. Look at the current version in master pom.xml